### PR TITLE
charset: comment a test to prevent it block ci

### PR DESCRIPTION
**Pull request by xiongjiwei**
### What problem does this PR solve?
close https://github.com/pingcap/tidb/issues/28099
this test will fail if update parser, comment it temporarily.

### Check List 

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by AilinKid — APPROVED** [review]

File: executor/executor_test.go [hunk 1]
```diff
@@ -5689,7 +5689,8 @@ func (s *testSuiteWithCliBaseCharset) TestCharsetFeature(c *C) {
 	tk.MustQuery("select @@character_set_connection;").Check(testkit.Rows("gbk"))
 	tk.MustQuery("select @@collation_connection;").Check(testkit.Rows("gbk_bin"))
 
-	tk.MustQuery("select _gbk 'a'").Check(testkit.Rows("a"))
+	// TODO: it will fail if update parser, comment it to prevent block CI. Will fix after https://github.com/pingcap/tidb/pull/27875 merged.
```
Comment (zimulala) [inline executor/executor_test.go:5692]: If we merge this PR, we will also get an error, right?

Comment (xiongjiwei) [reply executor/executor_test.go:5692]: yes, the error is expected. we will change the test to get an error

**Review by zimulala — APPROVED** [review]
LGTM

File: executor/executor_test.go (hunk 1 above)
Comment (AilinKid) [reply executor/executor_test.go:5692]: just skip here to unblock others, it will be fixed later

**Review by tangenta — APPROVED** [review]

**Pull request merged**